src/util/validateAuthToken.ts

```diff
@@ -0,0 +1,15 @@
+import { verify } from "jsonwebtoken";
+
+const validateAuthToken = (jwtToken: string) => {
+  
+  try {
+    if(jwtToken){
+      const userData = verify(jwtToken, process.env.JWTKEY);
+      if(typeof userData !== "string")
+      return {...userData.user}
```

Comment:
I find this code very difficult to read and I don't understand it's purpose.

* The indentation is wrong. The `return` statement will only run if the condition is true, but the indentation does not indicate that
* The return statement returns an object with all the attributes from `userData.user` (spread), which means that it could probably just `return userData.user` and have the same effect?
* In what cases does `verify` return a string? Doesn't it always return the payload or throw an error?